state/application.go

```diff
@@ -294,8 +294,9 @@ func (op *DestroyApplicationOperation) Done(err error) error {
 		rels, err2 := op.app.st.AllRelations()
 		if err2 != nil {
 			err = errors.Trace(err2)
+		} else {
```

Comment:
I am not sure that we need this additional else... Everything after the if- is the else-. What is the problem with the original code?

Reply:
Because `err = errors.Trace(err2)` becomes a no-op, as we set `err = errors.Errorf(` right after.